monkey/infection_monkey/monkey.py

```diff
@@ -86,6 +89,9 @@
 logger = logging.getLogger(__name__)
 logging.getLogger("urllib3").setLevel(logging.INFO)
 
+TMP_DIR = Path(gettempdir()) / random_filename()
+PLUGIN_DIR = TMP_DIR / "plugins"
```

Comment:
We'll need to actually create this directory (securely) at some point. Preferably in `_build_puppet()`.

Reply:
Yep, that's already being done! :-D
I see now that the parent directories are not automatically created, so I'll add that